addons/mail/static/src/rtc/call_action_list.js

```diff
@@ -1,15 +1,33 @@
 /* @odoo-module */
 
-import { Component } from "@odoo/owl";
+import { Component, useState } from "@odoo/owl";
 import { useRtc } from "@mail/rtc/rtc_hook";
 import { isMobileOS } from "@web/core/browser/feature_detection";
+import { Dropdown } from "@web/core/dropdown/dropdown";
+import { DropdownItem } from "@web/core/dropdown/dropdown_item";
+import { useService } from "@web/core/utils/hooks";
+import { _t } from "@web/core/l10n/translation";
 
 export class CallActionList extends Component {
+    static components = { Dropdown, DropdownItem };
     static props = ["thread", "fullscreen", "compact?"];
     static template = "mail.CallActionList";
 
     setup() {
         this.rtc = useRtc();
+        this.userSettings = useState(useService("mail.user_settings"));
+    }
+
+    get moreOptions() {
+        const options = [];
+        if (this.env.debug && this.userSettings.logRtc) {
```

Comment:
There are many ways to check if the app is in debug mode. I usually use `odoo.debug`, although I'm not sure if this is the best way to go.

Reply:
`env.debug` 6 occurrences in mail files vs 0 for `odoo.debug`. I think we should keep it as is for consistency sake